proto/burn/genesis.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package medibloc.panaceacore.burn;
+
+option go_package = "github.com/medibloc/panacea-core/x/burn/types";
+
+// GenesisState defines the burn module's genesis state.
+message GenesisState {
```

Comment:
`burn module`'s GenesisState is required. But deleted before PR.
So I rollbacked `genesis.proto` file.

removed commit here : https://github.com/medibloc/panacea-core/pull/122/commits/5c344e4a91924261577e672d46526b7ab5b7ae9b